site/content/contribute/getting-started/labels.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Labels"
+date: 2018-03-06T00:00:00-04:00
+weight: 5
+subsection: Getting Started
+---
+
+We leverage [GitHub labels](https://help.github.com/en/articles/about-labels) to track the details and lifecycle of issues and pull requests.
+
+# Issue Labels
+* `Area/<name>`: Involves changes to the named area (APIv4, Add E2E Tests, Localization, Plugins, etc.)
+* `Difficulty/1:easy`: Easy ticket.
+* `Difficulty/2:medium`: Medium ticket.
+* `Difficulty/3:hard`: Hard ticket.
+* `First Good Issue`: Suitable for first-time contributors.
+* `Framework/<name>`: Requires using the named framework (ReactJS, Redux, etc.)
+* `Help Wanted`: Community help wanted.
+* `Language/<name>`: Requires writing in the named language (Go, JavaScript, etc.)
+* `Move to Feature Ideas forum`: Marked for relocation to the feature ideas forum.
+* `Move to Troubleshooting`: Marked for relocation to the troubleshooting section of the documentation.
+* `PR Submitted`: A pull request has been opened for this issue.
+* `Repository/<name>`: Requires changes in the named respository.
```

Comment:
```suggestion

```

Reply:
This was removed - instead the repositories are listed in the help wanted issue description